modules/contact/test/tests/kernels/simple_diffusion/tests

```diff
@@ -1,7 +0,0 @@
-[Tests]
-  [./test]
-    type = 'Exodiff'
-    input = 'simple_diffusion.i'
-    exodiff = 'simple_diffusion_out.e'
-  [../]
-[]
```

Comment:
Umm... What's going on here?

Reply:
Do we really want our apps testing simple diffusion? Simple diffusion is done entirely with framework objects. We test that a zillion times every day, so by default it should work in an application.

Reply:
Didn't realize this was in the module. This is fine!

![image](https://user-images.githubusercontent.com/3036960/64461224-cf7b4400-d0b9-11e9-8605-4f9595748676.png)

Reply:
Ah - yeah we don't need this test here.

Reply:
Also a "zillion" is a bit of an exaggeration. It's more like a million: https://civet.inl.gov/num_tests/